app/models/repo.rb

```diff
@@ -147,16 +147,18 @@ def synchronize_pr_branches(expected)
 
   # TODO: Move this to GitService::Repo
   def git_fetch
-    require 'rugged'
-    rugged_repo = Rugged::Repository.new(path.to_s)
-    rugged_repo.remotes.each do |remote|
-      fetch_options = {}
+    # TODO: Figure out why this doesn't work on the bot using rugged
+    with_git_service { |git| git.fetch }
+    # require 'rugged'
+    # rugged_repo = Rugged::Repository.new(path.to_s)
+    # rugged_repo.remotes.each do |remote|
+    #   fetch_options = {}
 
-      username = extract_username_from_git_remote_url(remote.url)
-      fetch_options[:credentials] = Rugged::Credentials::SshKeyFromAgent.new(:username => username) if username
+    #   username = extract_username_from_git_remote_url(remote.url)
+    #   fetch_options[:credentials] = Rugged::Credentials::SshKeyFromAgent.new(:username => username) if username
 
-      rugged_repo.fetch(remote.name, fetch_options)
-    end
+    #   rugged_repo.fetch(remote.name, fetch_options)
+    # end
```

Comment:
Let's discuss before merge

Reply:
Okay
- I wanted to push this into master rather than rebasing the commit on the box every time.
- I usually don't like to comment code and leave it there (since it's in the git history), but I wanted to make it very obvious that this needs to be fixed.
- Also, I was hoping to get this in soon so that I can more easily update the box to the latest master

Reply:
Ah ok, so this is actually straight up broken and is the thing we have manually patched in the running bot.  Then I'm :+1: